### PR TITLE
Add EF Core based DataLoader sample with queries and mutation

### PR DESCRIPTION
**Pull request by huysentruitw**

### CODE REVIEW

File: samples/DataLoaderWithEFCore/DataLoaderWithEFCore/DataLoaderWithEFCore.csproj [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <ItemGroup>
+    <Compile Remove="wwwroot\**" />
+    <Content Remove="wwwroot\**" />
+    <EmbeddedResource Remove="wwwroot\**" />
+    <None Remove="wwwroot\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="7.0.1" />
+    <PackageReference Include="GraphQL.Conventions" Version="2.0.1" />
```
Comment (tlil) [inline samples/DataLoaderWithEFCore/DataLoaderWithEFCore/DataLoaderWithEFCore.csproj:16]: Could potentially use a project reference here?

File: samples/DataLoaderWithEFCore/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Start the project in Visual Studio and connect the GraphiQL client with the endpoint:
+
+    https://localhost:44329/api/graph
+
+### Query all data
+
+```
+query AllMovies {
+	movies {
```
Comment (tlil) [inline samples/DataLoaderWithEFCore/README.md]: Indentation #nitpick

File: samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{
+    public sealed class Actor
+    {
+        public Guid Id { get; set; }
+
+        public string Name { get; set; }
+
+        public string CountryCode { get; set; }
+
+        public Guid MovieId { get; set; }
```
Comment (tlil) [inline samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs:19]: Do you want this one in the schema? If you want to link to movie, perhaps having a `movies { ... }` field is more useful? :-)

**Review by tlil — COMMENTED** [review]
Neat - just a few minor comments

File: samples/DataLoaderWithEFCore/DataLoaderWithEFCore/DataLoaderWithEFCore.csproj (hunk 1 above)
Comment (huysentruitw) [reply samples/DataLoaderWithEFCore/DataLoaderWithEFCore/DataLoaderWithEFCore.csproj:16]: Do you want me to include the sample project in the main .sln file? (Only in that case a project reference would be better)

File: samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs (hunk 3 above)
Comment (huysentruitw) [reply samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs:19]: Yeah it's a bit an odd FK, but I wanted a 1 to many relation to demonstrate the use of a `CollectionBatchLoader`. Maybe we need to use a different use-case. Suggestions are welcome.

Comment (tlil) [reply samples/DataLoaderWithEFCore/DataLoaderWithEFCore/GraphApi/Schema/OutputTypes/Actor.cs:19]: Hmm, I see. Let's leave it like this for now.

**Pull request merged**